src/ScaffoldInstallerPlugin.php

```diff
@@ -72,6 +72,130 @@ public static function getSubscribedEvents()
         ];
     }
 
+    /**
+     * Composer configuration advice: Use local copies of patch files.
+     *
+     * @return void
+     */
+    private function _checkComposerPatchesAreLocal()
+    {
+        $patchesInComposer = $this->extra['patches'] ?? false;
+        $patchesInExtraFile = $this->extra['patches-file'] ?? false;
+
+        // Patches defined on a separate file.
+        if ($patchesInExtraFile) {
+            if (!file_exists($patchesInExtraFile)) {
+                $this->io->warning("The patches file `$patchesInExtraFile` can't be read.");
+                return;
+            }
+
+            $patchesJsonEncoded = file_get_contents($patchesInExtraFile);
+            $patchesContent = json_decode($patchesJsonEncoded, true)['patches'] ?? [];
```

Comment:
I think we need to either check `json_last_error()` or use an existing decoder utility function.

Reply:
Will do. Thanks!